SECP-service/src/main/java/com/visucius/secp/Controllers/User/UserProfileController.java

```diff
@@ -0,0 +1,105 @@
+package com.visucius.secp.Controllers.User;
+
+
+import com.google.common.base.Optional;
+import com.visucius.secp.DTO.UserDTO;
+import com.visucius.secp.models.User;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.visucius.secp.daos.UserDAO;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+
+public class UserProfileController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(UserProfileController.class);
+
+    private final UserDAO userDAO;
+
+    public UserProfileController(UserDAO userDAO){
+        this.userDAO = userDAO;
+    }
+
+    public Response getProfile(String userID){
+        long id = Long.parseLong(userID);
+        Optional<User> user = userDAO.find(id);
+
+        if(!user.isPresent()) {
+            throw new WebApplicationException(UserErrorMessage.USER_ID_INVALID, Response.Status.NO_CONTENT);
+        }
+        return Response.status(Response.Status.OK).entity(user.get()).build();
+    }
+
+    public void setDisplayName(UserDTO userDTO, long userID) {
+        validateDisplayName(userDTO);
+
+        String displayname = userDTO.getDisplayName();
+
+        Optional<User> user = userDAO.find(userID);
+        if(!user.isPresent()){
+            throw new WebApplicationException(UserErrorMessage.DISPLAY_NAME_FAILED_USER_NOT_FOUND, Response.Status.BAD_REQUEST);
+        }
+        User newuser = user.get();
+        newuser.setDisplayName(displayname);
+        userDAO.save(newuser);
+        LOG.info("Display name set.");
```

Comment:
you should return a response back to the client stating the change is successful. The response should have no body just a status code of 200.